src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/integration/SetupAssistant.kt

```diff
@@ -283,7 +290,15 @@ class SetupAssistant(
     val ppUser = createPPUser()
     val spUser = createSPUser()
     draftReferralRepository.save(referralFactory.createDraft(id = id, intervention = intervention, createdBy = ppUser, personCustodyPrisonId = personCustodyPrisonId, personCurrentLocationType = personCurrentLocationType, expectedReleaseDate = expectedReleaseDate))
-    return referralRepository.save(referralFactory.createEnded(id = id, intervention = intervention, createdBy = ppUser, sentBy = ppUser, endRequestedBy = ppUser, assignments = listOf(ReferralAssignment(OffsetDateTime.now(), spUser, spUser)), endRequestedReason = endRequestedReason, endRequestedComments = endRequestedComments))
+    return referralRepository.save(
+      referralFactory.createEnded(
+        id = id, intervention = intervention, createdBy = ppUser, sentBy = ppUser, endRequestedBy = ppUser,
+        assignments = listOf(
+          ReferralAssignment(OffsetDateTime.now(), spUser, spUser)
+        ),
+        endRequestedReason = endRequestedReason, endRequestedComments = endRequestedComments
```

Comment:
Another minor one, `endRequestedComments = endRequestedComments` can come in the next line